cli/cmd/generate_gcp.go

```diff
@@ -348,16 +353,53 @@ func initGenerateGcpTfCommandFlags() {
 		"bucket_lifecycle_rule_age",
 		-1,
 		"specify the lifecycle rule age")
-	generateGcpTfCommand.PersistentFlags().IntVar(
-		&GenerateGcpCommandState.LogBucketRetentionDays,
-		"bucket_retention_days",
-		0,
-		"specify the bucket retention days")
+	generateGcpTfCommand.PersistentFlags().StringVar(
+		&GenerateGcpCommandState.CustomFilter,
+		"custom_filter",
+		"",
+		"customer defined Audit Log filter which will supersede all other filter options when defined")
```

Comment:
```suggestion
		"customer-defined Audit Log filter which supersedes all other filter options when defined")
```

Reply:
I suggested hyphenating customer-defined, but I might make it user-defined instead or remove it altogether? 

```suggestion
		"Audit Log filter which supersedes all other filter options when defined")
```

Reply:
Thanks!  Just removing `customer defined`, does seem unnecessary.